### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_Eclipse Marketplace_Keyboard: The links "More info", "Microsoft Corp.", etc are not keyboard accessible

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#3363]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier!

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS2.1.1; #Keyboard; #C+E-H; #Java tooling for IntelliJ/Eclipse;

**Environment Details:**
Eclipse IDE for Enterprise Java Developers
Version: 2019-06 (4.12.0)
Build id: 20190614-1200

**OS Details:** 
OS: Windows 10 (Version 1903, Build: 18362.175)
MAS Violated: MAS 2.1.1

**Repro Steps: ​**
1) Open Eclipse application
2) Tab till Help and select "Eclipse Marketplace" from the list
3) Search/Find for 'Azure Toolkit for Eclipse' and press enter
4) Tab till the links present below "Azure Toolkit for Eclipse" heading and check if they are accessible through keyboard

**Actual Result: ​**
The links present below "Azure Toolkit for Eclipse" heading are not keyboard accessible 

**Expected Result: ​**
The links present below "Azure Toolkit for Eclipse" heading should be keyboard accessible 

**User Impact: ​**
Keyboard only users and screen reader users will not be provided with the information present in the links
 
**MAS Reference:**
[MAS 2.1.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2})

[Bug_3363_Eclipse_Marketplace_Links_Keyboard.pptx](https://github.com/microsoft/azure-tools-for-java/files/3451832/Bug_3363_Eclipse_Marketplace_Links_Keyboard.pptx)

**Comment by Flanker32** [comment]
Hi @soumyatippapuram, The eclipse marketplace is controlled by eclipse, which is out control of our plugin,  so could you please open an issue in eclipse(https://bugs.eclipse.org/bugs/). Here is a gif of search AWS in Eclipse marketplace, we could see links is not accessible in marketplace page.

![3363](https://user-images.githubusercontent.com/12445236/64938250-d7906c00-d88f-11e9-9aa2-04a5af56083a.gif)

**Comment by Flanker32** [comment]
* Importance of experience: Little, it's not a common behavior to access toolkit website in eclipse marketplace, we hadn't 
* Quality of experience: Little, screen reader could read the links when hover, and users could access the links in browser marketplace
* Alternate solutions: Users could visit Eclipse market place in browser and the links could be accessible with tab
* Evidence: All evidence is provided in this issue.

**Comment by Flanker32** [comment]
@soumyatippapuram This is an external issue which has been linked to AAT, please help us verify it, thanks!

**Comment by soumyatippapuram** [comment]
Reported this issue to Eclipse (https://bugs.eclipse.org/bugs/show_bug.cgi?id=551113)

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749764
